### DISCUSSION TITLE
Batch Select

### DISCUSSION THREAD

**Discussion by rolodexter** [AlexW00/obsidian-note-linker#16] [Ideas]
batch select
or select all

**Comment by AlexW00** [comment]
The best matches should be auto selected already.

**Comment by pleasedontspamme** [comment]
this is currently not the case on my system. Am I doing something wrong?

**Comment by Jamboc731** [comment]
I was just wondering if this has had any updates. I'm using this for character names in a diary style format and would like to link those names to their own notes. Currently on scanning the vault or the note, nothing is selected and I need to manually select each name occurrence to 'auto' link it

**Reply by Jamboc731** [reply]
Update: It works as intended if I add an Alias for all of the characters

**Comment by Type-IIx** [comment]
I came to report that after the initial scan to detect possible links, before beginning the link process, it displays all the possible links as checkboxes (multiple select) but these act instead as radio buttons (single select). I would like to link everything, if possible!
